src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController+Extension.swift

```diff
@@ -187,3 +195,24 @@ extension OnboardingInfoViewController {
 	}
 
 }
+
+
+extension OnboardingInfoViewController: UITextViewDelegate {
```

Comment:
We have a `// MARK: - Protocol <Protocol Name>` in our set of marks. Could you add the protocol conformances directly into the class and use the marks?

Reply:
New MARKs for extensions conforming to protocols